src/vscode-bicep/src/commands/build.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import vscode from "vscode";
+import { Command } from "./types";
+import { LanguageClient } from "vscode-languageclient/node";
+
+export class BuildCommand implements Command {
+  public readonly id = "bicep.build";
+  public readonly outputChannel =
+    vscode.window.createOutputChannel("Bicep Operations");
+
+  public constructor(private readonly client: LanguageClient) {}
+
+  public async execute(documentUri?: vscode.Uri | undefined): Promise<void> {
+    documentUri ??= vscode.window.activeTextEditor?.document.uri;
+
+    if (!documentUri) {
+      return;
+    }
+
+    const buildOutput: string = await this.client.sendRequest(
+      "workspace/executeCommand",
+      {
+        command: "build",
+        arguments: [documentUri.fsPath],
+      }
+    );
+
+    appendToOutputChannel(this.outputChannel, buildOutput);
```

Comment:
>appendToOutputChannel [](http://example.com/codeflow?start=4&length=21)

Btw, do we need to do something here so unhandled exceptions in the command get surfaced correctly in VS code?

My malformed JSON case should have thrown an exception in the language server, but nothing happened client side anywhere. When this happens with other LSP requests, you get a popup that the request failed.

Reply:
Added error handling logic here:
https://github.com/Azure/bicep/pull/3708#issuecomment-888676631

Added special case for malformed json to output below message:
![image](https://user-images.githubusercontent.com/30270536/127523187-8762aca4-5378-406d-a8df-697913f62a19.png)

Reply:
I think the popup gets tracked with telemetry, but failures in the output channel are not. I think we should stay consistent with the behavior for all other requests.

Reply:
As discussed offline, invoking client.error(..) to show popup.

Now that we are returning false for malformed json, I couldn't think of a good way to verify this. So I set context to null in debugger and verified popup shows up and redirects to bicep output channel. Here's the gif:

![Bicep_Build_5](https://user-images.githubusercontent.com/30270536/127549564-5e7912f3-5cd4-423e-9778-4d42e1789633.gif)